### PR TITLE
added research paper and a case study

### PR DESCRIPTION
**Pull request by sharad1126**
A new research paper has been added to the list of research papers related to IoT Security.

url: https://www.mdpi.com/1424-8220/19/19/4107

### CODE REVIEW

**Comment by sharad1126** [comment]
Also, added one case study on IoT Security@CERN - https://doi.org/10.5281/zenodo.1035034

File: readme.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ Software tools for analyzing embedded/IoT devices and firmware.
 
 <!--lint ignore match-punctuation-->
 
+- 2019, Agarwal et al: [Detecting IoT Devices and How They Put Large Heterogeneous Networks at Security Risk. Sensors 2019, 19, 4107.](https://www.mdpi.com/1424-8220/19/19/4107)
```
Comment (0xricksanchez) [inline readme.md:146]: Remove '. Sensors 2019, 19, 4107.' from the Title

File: readme.md [hunk 2]
```diff
@@ -165,6 +166,8 @@ Software tools for analyzing embedded/IoT devices and firmware.
 
 <!--lint ignore no-repeat-punctuation-->
 
+
```
Comment (0xricksanchez) [inline readme.md:169]: remove newline

File: readme.md [hunk 3]
```diff
@@ -165,6 +166,8 @@ Software tools for analyzing embedded/IoT devices and firmware.
...
+- [IoT Security@CERN](https://doi.org/10.5281/zenodo.1035034) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1035034.svg)](https://doi.org/10.5281/zenodo.1035034)
```
Comment (0xricksanchez) [inline readme.md:170]: Remove badge as the description. If you wish you can add a textual description. However make sure you'll finish the sentence with a '.'.

**Comment by 0xricksanchez** [comment]
I'll take a look at the two papers later today

File: readme.md (hunk 3 above)
Comment (0xricksanchez) [reply readme.md:170]: Also I'd like to keep the case studies in alphabetical order as for now

**Comment by sharad1126** [comment]
@0xricksanchez thanks.

**Pull request merged**